packages/web-sdk/src/instrumentations/performance/performanceUtils.ts

```diff
@@ -119,10 +120,22 @@ export function calculateFaroNavigationTiming(navigationEntryRaw: PerformanceNav
 
     type: type,
 
-    ...calculateFaroResourceTiming(navigationEntryRaw),
+    ...createFaroResourceTiming(navigationEntryRaw),
   };
 }
 
+function getDocumentParsingTime(domInteractive: number): string {
```

Comment:
nit: From a design PoV it makes sense to return the number with it being formatted with `toFaroPerformanceTimingString` so that it is consistent with the other attributes

Reply:
Very good hint. Let me change it 🙏